peachjam/resources.py

```diff
@@ -374,6 +375,18 @@ def clean(self, value, row=None, *args, **kwargs):
         return []
 
 
+class AttorneysWidget(ManyToManyWidget):
+    def clean(self, value, row=None, *args, **kwargs):
+
+        if value:
+            attorneys = [" ".join(j.split()) for j in value.split(self.separator)]
+
+            for attorney in attorneys:
+                obj, _ = self.model.objects.get_or_create(name=attorney)
+            return self.model.objects.filter(name__in=attorneys)
+        return []
```

Comment:
@nickmwangemi I think since this is doing the exact same thing as the judges widget, that we can reuse the Judges widget? Maybe rename it to something generic.